src/components/atoms/UserAvatar/UserAvatar.tsx

```diff
@@ -21,11 +21,12 @@ export const UserAvatar: React.FC<UserAvatarProps> = ({
 
   return (
     <div className={containerStyles}>
-      <img
+      <div
         onClick={onClick}
         className="user-avatar__image"
-        src={avatarSrc}
-        alt="user avatar"
+        style={{
+          backgroundImage: `url(${avatarSrc})`,
```

Comment:
Since `avatarSrc` can be potentially `undefined` here, we probably want to do something like the following (so it doesn't ender up rendering the styles as `url(undefined)`:

```suggestion
          backgroundImage: avatarSrc ? `url(${avatarSrc})` : undefined,
```